### DISCUSSION TITLE
How to upload a file as proof for school enrolment?

### DISCUSSION THREAD

**Discussion by AlphaHasher** [community/community#30135] [GitHub Education]
As the title says I can only see a button that allows me to take a picture with my webcam. I've tried to use a virtual webcam to feed a screenshot of a file to a webcam but I feel the resolution is not great so is it possible to just upload a file?

**Comment by Zoltus** [comment]
"Sometimes, for a number of reasons, our system needs additional levels of verification and asks that you take an image rather than upload one. Other suggestions might be to try applying while you are on campus, if able to."

Mayby if u try with device that doesnt have camera connected u have the upload button?

**Reply by AlphaHasher** [reply]
Sadly I don't see that option

**Reply by abdulrahman-hegazy** [reply]
@Zoltus can you help me over here
https://github.com/orgs/community/discussions/28390
I't would be a life changing opportunity

**Reply by mza0110** [reply]
@Zoltus can you help me with this one please https://github.com/orgs/community/discussions/30161
